src/site/markdown/server/manage-webadmin.md

```diff
@@ -58,6 +63,18 @@ Response codes:
  - 200: All checks have answered with a Healthy status
  - 500: At least one check have answered with a Unhealthy or Degraded status
 
+ ### List all health checks
```

Comment:
You have an extra space here which breaks markdown formating. Could you remove it?